node/resolvers/directives/helper.ts

```diff
@@ -1,8 +1,11 @@
+/* eslint-disable no-console */
+
 import { isUserPartOfBuyerOrg } from '../Queries/Users'
 
 export const validateAdminToken = async (
   context: Context,
-  adminUserAuthToken: string
+  adminUserAuthToken: string,
+  orgPermission?: 'buyer_organization_edit' | 'buyer_organization_view'
```

Comment:
sorry, my typescript is a bit rusty. What does this means exactly?
The `orgPermission` is optional, and should be either `buyer_organization_edit` or `buyer_organization_view`? Otherwise it will be empty? Is that it?

Reply:
Yes, if `org Permission` came, it should be `buyer_organization_edit` or `buyer organization view`, otherwise the property will be `undefined`